app/javascript/admin/displaySnackbar.js

```diff
@@ -0,0 +1,15 @@
+/**
```

Comment:
This function can be imported and used within any StimulusJS controller in Admin to show a Snackbar message.